### PR TITLE
#012 Feat: Create Footer Section

### PR DESCRIPTION
**Pull request by Fidesnoella**
# PR
## PR Description
Create footer section.
## Task ID and Ticket Card Link
Task ID: # 012
Ticket Card Link: [here](https://www.notion.so/apeunit/Footer-section-bdeb5f617ed24e6cb1d3ff2b43ba4350)
## Completed Tasks
- [x] create footer component
- [x] add footer component to homepage
## Testing
- npm install
- npm run dev
- view homepage

### CODE REVIEW

**Review by izzett222 — CHANGES_REQUESTED** [review]
on tablet the footer section is not divided into four equal parts, some part are bigger than the other in the design

File: tailwind.config.js [hunk 1]
```diff
@@ -54,12 +56,15 @@ module.exports = {
...
         2.5: "0.625rem", // 10px
         4.25: "1.0625rem", // 17px
+        10.5: "2.625rem", // 42px
         13.25: "3.3125rem", // 53px
         15.22: "3.8062rem", // 60.9px
         17.2: "4.3rem", // 68.8px
         17.9: "4.4938rem", // 71.9px
         18.4: "4.6rem", // 73.6px
         20.75: "5.1875rem", // 83px
+        1.25: "5.3125rem", //85px
```
Comment (izzett222) [inline tailwind.config.js]: 1.25 doesn't follow the convention of dividing the px value by four

File: components/layout/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            </div>
+            <div className="sm:border-l-0.5 border-black pl-0 sm:pl-5 pb-5 sm:pb-10  pr-4 sm:pr-2.5 pt-5 sm:pt-0.75 xl:pt-0">
+                <div className="max-w-39.5 sm:max-w-54.75">
+                    <Span>
+                        We are developers, designers, researchers and music enthusiast interested in building decentralised systems, fostering open knowledge and giving people tools to better coordinate.
+                    </Span>
+                </div>
+            </div>
+            <div className="flex flex-col justify-between border-l-0.5 border-black pl-5 sm:pl-2.5 xl:pl-5 pb-5 sm:pb-10 pt-0 sm:pt-0.75 xl:pt-0">
+                <span className=""><Anchor url={"mailto:even@apeunit.com"}>Get In Touch</Anchor></span>
```
Comment (izzett222) [inline components/layout/Footer.jsx]: Could you remove the empty className string?

File: components/layout/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            <div className="pb-0 sm:pb-10 pt-0.5 sm:pt-0.75 xl:pt-0 sm:pr-2 pr-5  xl:pl-10.5">
+                <h5 className="font-variation-secondary block font-semibold sm:text-sm text-xs -tracking-tight leading-3.75 sm:leading-4.25 text-black">Even</h5>
+                <Span>C/O Ape Unit,  Waldemarstr. 38</Span>
+                <Span>10999 Berlin Germany</Span>
+            </div>
+            <div className="border-l-0.5 border-black pb-0 sm:pb-10 pt-1 sm:pt-0.75 xl:pt-0 pr-2.5 xl:pr-21.25 pl-5 sm:pl-2.5 xl:pl-5 ">
+                <h5 className="font-variation-secondary block font-semibold sm:text-sm text-xs -tracking-tight leading-3.75 sm:leading-4.25 text-black">Miguel Lizio</h5>
+                <Span>Tel.: +49 30 306490108</Span>
+            </div>
+            <div className="sm:border-l-0.5 border-black pl-0 sm:pl-5 pb-5 sm:pb-10  pr-4 sm:pr-2.5 pt-5 sm:pt-0.75 xl:pt-0">
```
Comment (izzett222) [inline components/layout/Footer.jsx]: There is an extra space in the className. Could you remove it.

File: components/layout/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { Span } from "../ui/Typography";
+import Anchor from "../ui/Anchor";
+
+export default function Footer() {
+    return (
+        <div className="grid sm:grid-cols-4 grid-cols-2 mt-20 sm:mt-25">
+            <div className="pb-0 sm:pb-10 pt-0.5 sm:pt-0.75 xl:pt-0 sm:pr-2 pr-5  xl:pl-10.5">
```
Comment (izzett222) [inline components/layout/Footer.jsx]: There is an extra space in the className. Could you remove it?

File: components/layout/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+                <h5 className="font-variation-secondary block font-semibold sm:text-sm text-xs -tracking-tight leading-3.75 sm:leading-4.25 text-black">Even</h5>
+                <Span>C/O Ape Unit,  Waldemarstr. 38</Span>
+                <Span>10999 Berlin Germany</Span>
+            </div>
+            <div className="border-l-0.5 border-black pb-0 sm:pb-10 pt-1 sm:pt-0.75 xl:pt-0 pr-2.5 xl:pr-21.25 pl-5 sm:pl-2.5 xl:pl-5 ">
```
Comment (izzett222) [inline components/layout/Footer.jsx]: There is an extra space in the className at the end. Could you remove it.

File: components/layout/Footer.jsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { Span } from "../ui/Typography";
+import Anchor from "../ui/Anchor";
+
+export default function Footer() {
+    return (
+        <div className="grid sm:grid-cols-4 grid-cols-2 mt-20 sm:mt-25">
+            <div className="pb-0 sm:pb-10 pt-0.5 sm:pt-0.75 xl:pt-0 sm:pr-2 pr-5  xl:pl-10.5">
+                <h5 className="font-variation-secondary block font-semibold sm:text-sm text-xs -tracking-tight leading-3.75 sm:leading-4.25 text-black">Even</h5>
```
Comment (izzett222) [inline components/layout/Footer.jsx]: h5 tag is a block by default so the block is not needed.

File: components/layout/Footer.jsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+                <Span>C/O Ape Unit,  Waldemarstr. 38</Span>
+                <Span>10999 Berlin Germany</Span>
+            </div>
+            <div className="border-l-0.5 border-black pb-0 sm:pb-10 pt-1 sm:pt-0.75 xl:pt-0 pr-2.5 xl:pr-21.25 pl-5 sm:pl-2.5 xl:pl-5 ">
+                <h5 className="font-variation-secondary block font-semibold sm:text-sm text-xs -tracking-tight leading-3.75 sm:leading-4.25 text-black">Miguel Lizio</h5>
```
Comment (izzett222) [inline components/layout/Footer.jsx]: h5 tag is a block by default so the block is not needed.

**Review by izzett222 — CHANGES_REQUESTED** [review]

**Review by izzett222 — APPROVED** [review]

**Review by Nkaka23dev — APPROVED** [review]

**Pull request merged**